app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java

```diff
@@ -324,7 +323,7 @@ private void play(long position, int transcodedSubtitle) {
                 if (mVideoManager.isNativeMode()) mPlaybackState = PlaybackState.PLAYING; //won't get another onprepared call
                 if (mFragment != null) {
                     mFragment.setFadingEnabled(true);
-                    mFragment.setPlayPauseActionState(ImageButton.STATE_SECONDARY);
+                    mFragment.setPlayPauseActionState(0);
```

Comment:
Does the value here not matter? It looks like the SECONDARY state was 1 previously.

Reply:
Nope, the function never uses the argument.

Reply:
https://github.com/jellyfin/jellyfin-androidtv/blob/f668e1c66d19686c8692704ca90b2c253e1d204a/app/src/main/java/org/jellyfin/androidtv/ui/playback/CustomPlaybackOverlayFragment.java#L1337-L1339